variables.tf

```diff
@@ -42,6 +42,12 @@ variable "prevent_destroy" {
   default     = true
 }
 
+variable "key_destroy_scheduled_duration" {
+  description = "Set The period of time that versions of keys spend in the DESTROY_SCHEDULED state before transitioning to DESTROYED."
+  type        = string
+  default     = "86400s"
```

Comment:
Hmm, you mention an upcoming change to the API's default.  Would it work to use `null` here instead to allow for that change?  If the API instead returns an actual value such as `86400s`, that would unfortunately cause config churn on subsequent applies.

Reply:
That is a great point, didn't think of that. I'll update accordingly!

Reply:
and fixed